language-support/ts/codegen/tests/build-and-lint.sh

```diff
@@ -45,10 +47,14 @@ cp -rL $DAML_LEDGER/* $TMP_DAML_LEDGER
 
 cd $TMP_DIR
 
-$DAML2TS -o generated/src/daml $DAR
-sed -i "s/0.0.0-SDKVERSION/${VERSION}/" generated/package.json
+$DAML2TS -o daml2ts $DAR -p $TMP_DIR/package.json
 $YARN install --frozen-lockfile
-cd generated
-$YARN build
-$YARN lint
+# Remove the daml-types and daml-ledger workspaces. They were here for
```

Comment:
Unfortunately no. You want them in the workspaces at the `yarn install` step. That moves the packages into `node_modules`. They can't be in the workspaces for the subsequent `yarn workspaces run build` step though because that causes yarn to want to build `index.ts` in those packages which doesn't exist. This didn't come up before because we never ran `yarn workspaces build` rather, we would selectively `yarn run build` in directories in order.

Reply:
I figured out how to do it: use the `resolutions` field in the `package.json`. Here we go: https://github.com/digital-asset/daml/pull/4820.

Reply:
🎉